packages/yoroi-extension/app/api/ada/lib/storage/bridge/utils.js

```diff
@@ -329,11 +329,14 @@ export function isJormungandrAddress(
   kind: CoreAddressT
 ): boolean {
   // note: excluding legacy byron addresses
-  if (kind === CoreAddressTypes.JORMUNGANDR_SINGLE) return true;
-  if (kind === CoreAddressTypes.JORMUNGANDR_GROUP) return true;
-  if (kind === CoreAddressTypes.JORMUNGANDR_ACCOUNT) return true;
-  if (kind === CoreAddressTypes.JORMUNGANDR_MULTISIG) return true;
-  return false;
+  const types = [
+    CoreAddressTypes.JORMUNGANDR_SINGLE,
+    CoreAddressTypes.JORMUNGANDR_GROUP,
+    CoreAddressTypes.JORMUNGANDR_ACCOUNT,
+    CoreAddressTypes.JORMUNGANDR_MULTISIG,
+  ];
+
+  return kind in types;
```

Comment:
@AhmedIbrahim336 Please note that this change is buggy. The `in` operator can't be used to test array membership. I'll revert this change.